assemble/bin/accumulo-cluster

```diff
@@ -467,10 +469,17 @@ function control_services() {
     done
   fi
 
-  if [[ $ARG_LOCAL == 0 && $ARG_ALL == 1 && ($operation == "stop" || $operation == "kill") ]]; then
-    if ! isDebug; then
-      echo "Cleaning all server entries in ZooKeeper"
-      $accumulo_cmd org.apache.accumulo.server.util.ZooZap -manager -tservers -compactors -sservers
+  if [[ $ARG_LOCAL == 0 && ($operation == "stop" || $operation == "kill") ]]; then
+    if [[ $ARG_TSERVER == 1 ]]; then
+      if ! isDebug; then
+        echo "Cleaning all tserver entries in ZooKeeper"
+        $accumulo_cmd org.apache.accumulo.server.util.ZooZap -tservers -group "$zap_group"
```

Comment:
This needs to be quoted because it can contain spaces in the path.

```suggestion
        "$accumulo_cmd" org.apache.accumulo.server.util.ZooZap -tservers -group "$zap_group"
```